Add ``osu! Philippines Nationals 2024 - osu!mania 4K/7K`` tournament article (#11985)
ppy/osu-wiki: wiki/Tournaments/oPHN/2024_mania/en.md

---
tags:
  - PHNM2024
  - PHNM 2024
---

# osu! Philippines Nationals 2024 - osu!mania 4K/7K

![PHNM2024 logo](img/logo.jpg)

The **osu! Philippines Nationals 2024 - osu!mania 4K/7K** Tournament (***PHNM2024***) was a 1v1 osu!mania 4-key and 7-key tournament. It serves as the National Championships of both osu!mania keymodes for the Philippine community of the game.

## Tournament Schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2024-04-20/2024-05-05 (15:59 UTC) |
| Qualifiers showcase | 2024-05-25 (13:00 UTC) |
| Qualifiers | 2024-06-01/2024-06-02 |
| Round of 16 | 2024-06-08/2024-06-09 |
| Quarterfinals | 2024-06-15/2024-06-16 |
| Semifinals | 2024-06-22/2024-06-23 |
| Finals | 2024-06-29/2024-06-30 |
| Grand Finals | 2024-07-06/2024-07-07 |

## Prizes

The winners of each division will receive a profile badge.

![](img/phnm-badge_4k.png "PHNM 4K winner badge")
![](img/phnm-badge_7k.png "PHNM 7K winner badge")

## Organization

> The osu! Philippines Nationals 2024 - osu!mania 4K/7K tournament is ran by various community members.

Reviewer: ```suggestion
The osu! Philippines Nationals 2024 - osu!mania 4K/7K tournament was run by various community members.
```
